### PR TITLE
Document compatability API limits

### LINKED ISSUE #1424: Mention limitations of 1.x compatibility
_Describe the issue here._

There are several limitations of InfluxDB 1.x compatibility in InfluxDB open source and Cloud that are not mentioned in the docs.

- The `INTO` clause, e.g. `SELECT ... INTO ...`, is not supported.
- None of the [InfluxQL database management commands](https://docs.influxdata.com/influxdb/v1.8/query_language/manage-database/#delete-series-with-delete) are supported.
    - Except `DELETE` and `DROP MEASUREMENT` queries.

##### Relevant URLs
- https://docs.influxdata.com/influxdb/v2.0/reference/api/influxdb-1x/

##### What products and version are you using?
InfluxDB Cloud

### PR DESCRIPTION
**Pull request by pierwill**
Clarify compatibility API support for InfluxQL.

Closes #1424.

- [ ] Tests pass (no build errors)
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/reference/api/influxdb-1x/_index.md [hunk 1]
```diff
@@ -57,6 +57,13 @@ Authorization: Token <token>
 Authorization: Token mYSuP3rs3cREtT0k3N
 ```
 
+##### InfluxQL support
+
+The compatability API supports InfluxQL, with the following caveats:
+
+- The `INTO` clause (e.g. `SELECT ... INTO ...`) is not supported.
+- With the exception of `DELETE` and `DROP MEASUREMENT` queries, which are still allowed, InfluxQL database management commands are not supported.
```
Comment (gunnaraasen) [inline content/influxdb/v2.0/reference/api/influxdb-1x/_index.md:65]: Let's add some links to the relevant pages in the InfluxQL docs?

File: content/influxdb/v2.0/reference/api/influxdb-1x/_index.md [hunk 2]
```diff
@@ -57,6 +57,13 @@ Authorization: Token <token>
 Authorization: Token mYSuP3rs3cREtT0k3N
 ```
 
+##### InfluxQL support
+
+The compatability API supports InfluxQL, with the following caveats:
+
+- The `INTO` clause (e.g. `SELECT ... INTO ...`) is not supported.
```
Comment (gunnaraasen) [inline content/influxdb/v2.0/reference/api/influxdb-1x/_index.md:64]: For management commands and moving data, I think we should also mention and link to how it's possible to do the same tasks with the v2 API.

**Review by gunnaraasen — CHANGES_REQUESTED** [review]
Thanks @pierwill! Just a couple suggestions, otherwise LGTM

File: content/influxdb/v2.0/reference/api/influxdb-1x/_index.md (hunk 2 above)
Comment (pierwill) [reply content/influxdb/v2.0/reference/api/influxdb-1x/_index.md:64]: @gunnaraasen Would it make sense to link to https://docs.influxdata.com/influxdb/v2.0/reference/cli/influx/bucket/delete/ and https://docs.influxdata.com/influxdb/v2.0/reference/cli/influx/delete/?

**Review by gunnaraasen — APPROVED** [review]

**Pull request merged**